bigchaindb/commands/bigchain.py

```diff
@@ -122,8 +146,12 @@ def run_drop(args):
 
 def run_start(args):
     """Start the processes to run the node"""
-    # run_configure(args, skip_if_exists=True)
     bigchaindb.config_utils.autoconfigure(filename=args.config, force=True)
+
+    if args.start_rethinkdb:
```

Comment:
no need, `start_rethinkdb` will fail because [this](https://github.com/bigchaindb/bigchaindb/pull/224/files#diff-8f049242225a1f66c973d24911b5179aR22) won't happen.